plugins/browser-plugin-webview/src/index.ts

```diff
@@ -0,0 +1,70 @@
+import { BrowserPlugin, BrowserTracker, Payload } from '@snowplow/browser-tracker-core';
+import { hasMobileInterface, trackWebViewEvent } from '@snowplow/webview-tracker';
+import { Logger, SelfDescribingEvent, SelfDescribingJson } from '@snowplow/tracker-core';
+
+const _trackers: Record<string, BrowserTracker> = {};
+
+export function WebViewPlugin(): BrowserPlugin {
+  let LOG: Logger;
+
+  return {
+    activateBrowserPlugin: (tracker: BrowserTracker) => {
+      _trackers[tracker.id] = tracker;
+    },
+
+    logger: (logger: Logger) => {
+      LOG = logger;
+    },
+
+    filter: (payload) => {
+      if (hasMobileInterface() === true) {
+        LOG.debug(`Payload (event ID: ${payload.eid}) was filtered out and forwarded to WebView tracker.`);
+
+        let atomicProperties = {
+          eventName: payload.e as string,
+          trackerVersion: payload.tv as string,
+          useragent: payload.ua as string,
```

Comment:
I think by default we don't set the `ua` property in events on the JS tracker since that is parsed from the HTTP headers by collector. However, since in the WebView tracker, we won't be making the request directly from the browser, we will need to pass them manually.

So I'd suggest that we add a default value here – in case `payload.ua` is not present, we should read the useragent from `window.navigator.userAgent`.